docs/provider/overview.md

```diff
@@ -0,0 +1,34 @@
+# Provider overview
+
+## Installation
+
+**Default installation:**
+
+`curl -sSf https://join.golem.network/as-provider | bash -`
+
+**Installing specific version:**
+
+`curl -sSf https://join.golem.network/as-provider | YA_INSTALLER_CORE=pre-rel-v0.13.0-rc10 bash -`
+
+`YA_INSTALLER_CORE` should point to tag in yagna repository.
+
+To change default runtimes versions set env variables:
+
+- `YA_INSTALLER_WASI=v0.2.2`
+- `YA_INSTALLER_VM=v0.3.0`
+
+## Provider directories
+
+Standard Provider installation uses files in following directories:
+
+
+| name                    | ENV           | command line                | default configuration                     | description                                                                                                                                                                                                            | comment                                                                      |
+| ----------------------- | ------------- | --------------------------- | ----------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------- |
+| Yagna data directory    | YAGNA_DATADIR | yagna --datadir             | ~/.local/share/yagna                      | Contains yagna daemon configuration and persistent files.                                                                                                                                                              |                                                                              |
+| Provider data directory | DATADIR       | ya-provider --datadir       | ~/.local/share/ya-provider                | Provider agent configuration files, logs and ExeUnit directories.                                                                                                                                                      |                                                                              |
+| Runtimes directory      | EXE_UNIT_PATH | ya-provider --exe-unit-path | ~/.local/lib/yagna/plugins/ya-*.json      | Contains runtime binaries.                                                                                                                                                                                             | Regular expression pointing to ExeUnits descriptors (It's not directory).    |
```

Comment:
There is still an issue with this env variable getting override by `golemsp` 
https://github.com/golemfactory/yagna/blob/master/golem_cli/src/command.rs#L46

Reply:
I created an issue for it https://github.com/golemfactory/yagna/issues/2689

Reply:
In next PR I will add comment that this is how golemsp handles this